blocks.css

```diff
@@ -32,7 +33,13 @@ ol,
 .wp-block-cover-text.alignfull,
 .wp-block-image.alignfull,
 .wp-block-gallery.alignfull,
-..wp-block-text-columns.alignfull{
+.wp-block-text-columns.alignfull{
   margin: 1.5em 0;
   max-width: 100%;
 }
+
+.wp-block-button.aligncenter {
+	display: block;
+	margin:0 auto;
```

Comment:
More indentation here than in the rest of the stylesheet — maybe swap out to match the rest of the stylesheet?